common/constants.go

```diff
@@ -45,6 +45,7 @@ var TIPXDCX = big.NewInt(38383838)
 var TIPXDCXLending = big.NewInt(38383838)
 var TIPXDCXCancellationFee = big.NewInt(38383838)
 var TIPXDCXCancellationFeeTestnet = big.NewInt(38383838)
+var TIPXDCXDISABLE = big.NewInt(99999999900)
```

Comment:
right, so you are putting a placeholder for now? what's the plan on the actual disable time?

Reply:
ah, sorry. i think u just try to test it out on dev and testnet first